php56/root/etc/confd/templates/xdebug.ini.tmpl

```diff
@@ -8,12 +8,15 @@ xdebug.coverage_enable=0
 xdebug.default_enable=1
 
 xdebug.remote_enable=1
+xdebug.remote_autostart=1
 xdebug.remote_connect_back=1
-xdebug.remote_host=localhost
+; If the following remote_host value does not work use the environmental
+; variable XDEBUG_CONFIG to override it. Documentation for format at
+; https://xdebug.org/docs/remote
+xdebug.remote_host=192.168.99.1
 xdebug.remote_port=9000
```

Comment:
I like the idea of documenting the XDEBUG_CONFIG variable in the readme as the method to override these but I like having them in the config file so that someone who wants to override one only has to concern themselves with the ones that they want to override rather than needing to ensure they maintain all the other values. Will ship through an update to the README shortly.